browser/main/NoteList/index.js

```diff
@@ -687,8 +691,13 @@ class NoteList extends React.Component {
         {
           label: copyNoteLink,
           click: this.copyNoteLink.bind(this, note)
+        },
+        {
+          label: openInNewWindow,
+          click: this.openInNewWindow.bind(this)(note)
```

Comment:
Shouldn't it be like this?
```suggestion
          click: this.openInNewWindow.bind(this, note)
```